.github/linters/.markdown-lint.yml

```diff
@@ -0,0 +1,4 @@
+{
+  "MD013": false,
+  "MD033": false
```

Comment:
this is needed because some of the text in a design doc is <spec> which isn't html but markdown lint thinks it is.